Purchase Order | test_po_with_pricing_rule_TC_B_046 |     self.assertEqual(doc_po_item.rate, 117)
AssertionError: 7359.3 != 117

### Similar Testcases:- Failed
test_po_with_pricing_rule_TC_B_047

### Issue Details:
    self.assertEqual(doc_po_item.rate, 117)
AssertionError: 7359.3 != 117

### Traceback:
``` 
FAIL: test_po_with_pricing_rule_TC_B_046 (erpnext.buying.doctype.purchase_order.test_purchase_order.TestPurchaseOrder)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/frappe/test-postgres/test-bench/apps/erpnext/erpnext/buying/doctype/purchase_order/test_purchase_order.py", line 2585, in test_po_with_pricing_rule_TC_B_046
    self.assertEqual(doc_po_item.rate, 117)
AssertionError: 7359.3 != 117
```

--- a/erpnext/selling/doctype/sales_order_item/sales_order_item.json
+++ b/erpnext/selling/doctype/sales_order_item/sales_order_item.json
@@ -109,8 +109,7 @@
   "purchase_order_item",
   "accounting_dimensions_section",
   "cost_center",
-  "column_break_ihdh",
-  "project"
+  "column_break_ihdh"
  ],
  "fields": [
   {
@@ -912,59 +911,51 @@
    "label": "Is Stock Item",
    "print_hide": 1,
    "report_hide": 1
- },
- {
-  "allow_on_submit": 1,
-  "fieldname": "company_total_stock",
-  "fieldtype": "Float",
-  "label": "Qty (Company)",
-  "no_copy": 1,
-  "print_hide": 1,
-  "read_only": 1
- },
- {
-  "fieldname": "column_break_jpky",
-  "fieldtype": "Column Break"
- },
- {
-  "fieldname": "available_quantity_section",
-  "fieldtype": "Section Break",
-  "label": "Available Quantity"
-},
-{
- "collapsible": 1,
- "fieldname": "accounting_dimensions_section",
- "fieldtype": "Section Break",
- "label": "Accounting Dimensions"
-},
-{
- "allow_on_submit": 1,
- "default": ":Company",
- "fieldname": "cost_center",
- "fieldtype": "Link",
- "label": "Cost Center",
- "options": "Cost Center",
- "print_hide": 1,
- "print_width": "120px",
- "width": "120px"
-},
-{
- "fieldname": "column_break_ihdh",
- "fieldtype": "Column Break"
-},
-{
- "allow_on_submit": 1,
- "fieldname": "project",
- "fieldtype": "Link",
- "label": "Project",
- "options": "Project",
- "search_index": 1
- }
+  },
+  {
+   "allow_on_submit": 1,
+   "fieldname": "company_total_stock",
+   "fieldtype": "Float",
+   "label": "Qty (Company)",
+   "no_copy": 1,
+   "print_hide": 1,
+   "read_only": 1
+  },
+  {
+   "fieldname": "column_break_jpky",
+   "fieldtype": "Column Break"
+  },
+  {
+   "fieldname": "available_quantity_section",
+   "fieldtype": "Section Break",
+   "label": "Available Quantity"
+  },
+  {
+   "collapsible": 1,
+   "fieldname": "accounting_dimensions_section",
+   "fieldtype": "Section Break",
+   "label": "Accounting Dimensions"
+  },
+  {
+   "allow_on_submit": 1,
+   "default": ":Company",
+   "fieldname": "cost_center",
+   "fieldtype": "Link",
+   "label": "Cost Center",
+   "options": "Cost Center",
+   "print_hide": 1,
+   "print_width": "120px",
+   "width": "120px"
+  },
+  {
+   "fieldname": "column_break_ihdh",
+   "fieldtype": "Column Break"
+  }
  ],
  "idx": 1,
  "istable": 1,
  "links": [],
- "modified": "2025-02-06 13:29:24.619850",
+ "modified": "2025-03-28 17:57:30.556452",
  "modified_by": "Administrator",
  "module": "Selling",
  "name": "Sales Order Item",

--- a/erpnext/selling/doctype/sales_order_item/sales_order_item.py
+++ b/erpnext/selling/doctype/sales_order_item/sales_order_item.py
@@ -69,7 +69,6 @@ class SalesOrderItem(Document):
 		pricing_rules: DF.SmallText | None
 		produced_qty: DF.Float
 		production_plan_qty: DF.Float
-		project: DF.Link | None
 		projected_qty: DF.Float
 		purchase_order: DF.Link | None
 		purchase_order_item: DF.Data | None

